v2/charts/azure-service-operator/values.yaml

```diff
@@ -92,6 +92,9 @@ image:
 # 'address' field defines the metrics binding address on which metrics
 metrics:
   enable: true
+  # secureMetrics controls whether metrics should be served via 'http' or 'https'.
```

Comment:
It's more than just HTTP vs HTTPS - if you use secure mode, it means that we will also be doing RBAC checks. Should we explain that here?

Reply:
Yes you already document the required roles in the `metrics.md` - we can possibly link to it here, OR just include a little snippet w/ the names of the roles (or both).